tests/calculate.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+cd $1
+
+find .  -name "*\.zip" -exec unzip -q {} \; >/dev/null # unzip the results files
+
+# - .txt files have no stochastic content, may be md5sum-checked
+# - .html files are generically named, no stochastic content, may be md5sum-checked
+# - .fo files are generically named, no stochastic content, may be md5sum-checked
+# - Icons/*.png files are also generically named, no stochastic content, may be md5sum-checked
+# - Images/*.png - the same
+
+# Therefore:
+# - Check md5sums for all types of files, sort
+
+echo ".txt files:"
+find . -name "*.txt" | xargs md5sum | sort -V
+
+echo ".html files:"
+find . -name "*.html" | xargs md5sum | sort -V
```

Comment:
When I was previously trying to compare fastqc outputs, I found that there's a report generation date baked into the header. In the output files generated by the test it looks like this:
```
<div id="header_filename">Mon 8 Nov 2021<br/>$SAMPLE_NAME</div>
```
Could the date be stripped from the file before calculating the md5sum?

Reply:
Yes, of course!